### ISSUE TITLE
gcloudignore didn't work when deploying API to Cloud Run using GitHub Actions

### ISSUE THREAD

**Issue by ajipandean** [google-github-actions/setup-gcloud#468]
### TL;DR

Google Cloud Run cannot read .env. staging file when deployed using GitHub Actions, whereas I've included the .env.staging file in .gcloudignore.

This is my .gcloudignore:
```
.gcloudignore
.git

!.env.staging
!service-account.json
```

### Expected behavior

The .env.staging file should be exist in Cloud Run container and my API should be able to read it.

### Observed behavior

My API cannot read the .env.staging file and caused deployment error when using Github Actions
![Screenshot from 2022-01-04 14-29-25](https://user-images.githubusercontent.com/37402263/148020392-52acc596-2085-492a-ae0d-4efa3c77d173.png)
![Screenshot from 2022-01-04 14-28-39](https://user-images.githubusercontent.com/37402263/148020406-9799ad77-f312-4e09-954b-fc23fe2193f4.png)

### Action YAML

```yaml
name: CI/CD to Google Cloud Run

on:
  release:
    types:
      - published

jobs:
  main:
    name: Continous Deployment to Google Cloud Run
    runs-on: ubuntu-latest
    permissions:
      contents: read
      id-token: write

    steps:
      - uses: actions/checkout@v2

      - id: auth
        name: Authenticate service account
        uses: google-github-actions/auth@v0
        with:
          credentials_json: ${{secrets.SERVICE_ACCOUNT_JSON}}

      - id: gcloud
        name: Setup Google Cloud SDK
        uses: google-github-actions/setup-gcloud@v0
        with:
          project_id: ${{secrets.GCP_PROJECT_ID}}

      - id: deploy
        name: Deploy API to Google Cloud Run
        run: gcloud builds submit --config deployments/cloudbuild.staging.yaml
```

### Additional information

I don't know it's a bug or not, but when I deployed my API using gcloud command line locally, it worked fine. The error only happen when using Github Actions.

Any solutions?

Thanks.

**Comment by sethvargo** [comment]
Is `.env.staging` checked into the source repository?

**Comment by ajipandean** [comment]
Do you mean "is ```.env.staging``` pushed into the source repository?"? No, it isn't.

It ignored by git.

**Comment by sethvargo** [comment]
Then it wouldn't work with GitHub Actions. GitHub Actions clones the repo from GitHub.

**Comment by ajipandean** [comment]
Ahh I see. How can I didn't notice that thing :D

Is there any possible solution that I can implement?

**Comment by sethvargo** [comment]
You would need to either check that value into source control or, if they values are sensitive, set them in GitHub Secrets.
